docs/swagger.yaml

```
@@ -421,33 +421,94 @@ definitions:
       refresh_token:
         type: string
     type: object
-  domain.UnauthorizedError:
+  domain.SaveUnauthAppointmentRequest:
```

YAMLlint (1.38.0)

[error] duplication of key "domain.SaveUnauthAppointmentRequest" in mapping

(key-duplicates)